UI/GameInfoCache.cpp

```diff
@@ -229,6 +229,11 @@ bool GameInfo::LoadFromPath(const std::string &gamePath) {
 }
 
 std::shared_ptr<FileLoader> GameInfo::GetFileLoader() {
+	if (filePath_.empty()) {
+		// Happens when workqueue tries to figure out priorities in PrioritizedWorkQueue::Pop(),
+		// because priority() calls GetFileLoader()... gnarly.
+		return fileLoader;
```

Comment:
Just remember that for some loaders, e.g. ones that involve memory caching or http access, the cost of tossing this in the bin and getting a new one a couple times is much higher.  Ideally we shouldn't recreate just because that's easiest.

-[Unknown]

Reply:
Oops, put this comment in the wrong place somehow - was meant for the lifetime comment.

-[Unknown]